src/parser/jobs/whm/modules/Overheal.js

```diff
@@ -1,78 +1,27 @@
-import STATUSES from 'data/STATUSES'
-import Module from 'parser/core/Module'
-import {TieredRule, TARGET, Requirement} from 'parser/core/modules/Checklist'
-import {Trans} from '@lingui/react'
 import React from 'react'
+import {Trans} from '@lingui/macro'
```

Comment:
Any reason you user macro vs react for this import?

Reply:
Honestly, I think when I want to remember what it is, I look at someone else's module (because I can never remember what it's called).

What's the difference, out of curiosity? If there's a reason for one over the other, I'll change it.

Reply:
Went with `react` also after talking with Ack.